charts/pxc-db/templates/s3-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- range $key, $value := .Values.backup.storages }}
+{{- if and (hasKey $value "type") (eq $value.type "s3") (hasKey $value "s3") (hasKey (index $value "s3") "credentialsAccessKey") (hasKey (index $value "s3") "credentialsSecretKey") }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "pxc-database.fullname" $ }}-s3-{{ $key }}
+  labels:
+{{ include "pxc-database.labels" $ | indent 4 }}
+type: Opaque
+data:
+  AWS_ACCESS_KEY_ID: {{ index $value "s3" "credentialsAccessKey" | b64enc }}
+  AWS_SECRET_ACCESS_KEY: {{ index $value "s3" "credentialsSecretKey" | b64enc }}
+{{- end }}
```

Comment:
Thanks for nice contribution! one comment, please add a delimiter if we have for example 2 secrets with these values, like so:
```
    s3-us-west:
      type: s3
      s3:
        bucket: aaaaaaaaaaaaaaaaaaaaaaaa
        #        credentialsSecret: my-cluster-name-backup-s3
        credentialsAccessKey: aaa
        credentialsSecretKey: bbb
        region: us-west-2
        endpointUrl: https://sfo2.digitaloceanspaces.com
    s3-us-west-binlog:
      type: s3
      s3:
        bucket: bbbbbbbbbbbbbbbbbbbbbbbb
        #        credentialsSecret: my-cluster-name-backup-s3
        credentialsAccessKey: bbbbbb
        credentialsSecretKey: cccccc
        region: us-west-2
        endpointUrl: https://sfo2.digitaloceanspaces.com
```
they end up glued together without delimiter between them, so I don't think it will work properly in this case:
```
---
# Source: pxc-db/templates/s3-secret.yaml
apiVersion: v1
kind: Secret
metadata:
  name: production-s3-s3-us-west
  labels:
    app.kubernetes.io/name: production
    helm.sh/chart: pxc-db-0.1.14
    app.kubernetes.io/instance: cluster1
    app.kubernetes.io/version: "1.6.0"
    app.kubernetes.io/managed-by: Helm
type: Opaque
data:
  AWS_ACCESS_KEY_ID: YWFh
  AWS_SECRET_ACCESS_KEY: YmJi
apiVersion: v1
kind: Secret
metadata:
  name: production-s3-s3-us-west-binlog
  labels:
    app.kubernetes.io/name: production
    helm.sh/chart: pxc-db-0.1.14
    app.kubernetes.io/instance: cluster1
    app.kubernetes.io/version: "1.6.0"
    app.kubernetes.io/managed-by: Helm
type: Opaque
data:
  AWS_ACCESS_KEY_ID: YmJiYmJi
  AWS_SECRET_ACCESS_KEY: Y2NjY2Nj
---
```
also you'll need to bump the chart version to `0.1.16`

Thanks!

Reply:
Good catch, thanks! :) I've updated the code with appropriate YAML delimiter.